docs/samples/management-api-v2-walkthrough/2-assets.md

```diff
@@ -0,0 +1,83 @@
+# Creating an Asset
+
+## Old plain JSON Schema
```

Comment:
do we need the old example? it could be misleading. same for all the other markdowns

Reply:
personally, i always appreciate those old-vs-new comparisons in readmes, as long as it is made very clear which one is the one to use. but i'm fine with both keeping and removing.